examples/mergable-ingress-types/README.md

```diff
@@ -0,0 +1,105 @@
+host level, which includes the TLS configuration, and any annotations which will be applied for the complete host. There
+can only be one ingress resource on a unique host that contains the master value. Paths cannot be part of the
+ingress resource.
+
+A Minion is declared using `nginx.org/mergible-ingress-type: minion`. A Minion will be used to append different
+locations to an ingress resource with the Master value. Only annotations which contain the serviceName key can be used
+here. TLS configurations and annotations which apply to the complete host are not allowed. There can be multiple minions
+which must have the same host as the master.
+
+Note: Ingress Resources with more than one host cannot be used.
+
+## Example
+
+In this example we deploy the NGINX or NGINX Plus Ingress controller, a simple web application and then configure
+load balancing for that application using Ingress resources with the `nginx.org/mergible-ingress-type` annotations.
+
+## Running the Example
+
+## 1. Deploy the Ingress Controller
+
+1. Follow the installation [instructions](../../docs/installation.md) to deploy the Ingress controller.
+
+2. Save the public IP address of the Ingress controller into a shell variable:
+    ```
+    $ IC_IP=XXX.YYY.ZZZ.III
+    ```
+
+## 2. Deploy the Cafe Application
+
+Create the coffee and the tea deployments and services:
+```
+$ kubectl create -f cafe.yaml
+```
+
+## 3. Configure Load Balancing
+
+1. Create a secret with an SSL certificate and a key:
+    ```
+    $ kubectl create -f cafe-secret.yaml
+    ```
+
+2. Create the Master Ingress resource:
+    ```
+    $ kubectl create -f cafe-master.yaml
+    ```
+    
+3. Create the Minion Ingress resource for the Coffee Service:
+    ```
+    $ kubectl create -f coffee-minion.yaml
+    ```
+
+4. Create the Minion Ingress resource for the Tea Service:
+   ```
+    $ kubectl create -f tea-minion.yaml
+    ```
+
+## 4. Test the Application
+
+1. To access the application, curl the coffee and the tea services. We'll use ```curl```'s --insecure option to turn off certificate verification of our self-signed
+certificate and the --resolve option to set the Host header of a request with ```cafe.example.com```
+    
+    To get coffee:
+    ```
+    $ curl --resolve cafe.example.com:443:$IC_IP https://cafe.example.com/coffee --insecure
+    Server address: 10.12.0.18:80
+    Server name: coffee-7586895968-r26zn
+    ...
+    ```
+    If you prefer tea:
+    ```
+    $ curl --resolve cafe.example.com:443:$IC_IP https://cafe.example.com/tea --insecure
+    Server address: 10.12.0.19:80
+    Server name: tea-7cd44fcb4d-xfw2x
+    ...
+    ```
+
+    **Note**: If you're using a NodePort service to expose the Ingress controller, replace port 443 in the commands above with the node port that corresponds to port 443.
+
+2. If you're using NGINX Plus, you can open the live activity monitoring dashboard:
```

Comment:
Could you remove the part about NGINX Plus?

Reply:
Done